eq-author/src/components/CommentAccordion/index.test.js

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import { render, fireEvent, act } from "tests/utils/rtl";
+import CommentAccordion from "./";
+
+describe("CommentAccordion", () => {
+  it("should render open", () => {
+    const { getByTestId } = render(
+      <CommentAccordion title="1">Comment Accordion panel</CommentAccordion>
+    );
+    expect(getByTestId("accordion-1-body")).not.toBeVisible();
+    expect(getByTestId("accordion-1-button")).toBeVisible();
+  });
+
+  it("should open and close accordion", async () => {
+    const { getByTestId } = render(
+      <CommentAccordion title="1">Comment Accordion panel</CommentAccordion>
+    );
+    expect(getByTestId("accordion-1-body")).not.toBeVisible(); //close
+
+    await act(async () => {
+      await fireEvent.click(getByTestId("accordion-1-button"));
+    });
+    expect(getByTestId("accordion-1-body")).toBeVisible(); //open
+
+    await act(async () => {
+      await fireEvent.click(getByTestId("accordion-1-button"));
+    });
+    expect(getByTestId("accordion-1-body")).not.toBeVisible(); //close
```

Comment:
We shouldn't really need to have comments in tests, I feel. I think it's understandable without them, but if you think they're needed, maybe consider splitting the open and close tests into two separate tests.